index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Product Landing Page</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+        <header id="header">
+            <img src="logo.png" alt="" id="header-img">
+            <nav id="nav-bar">
+                <a href="#header" class="nav-link">Home</a>
+                <a href="#about" class="nav-link">About</a>
+                <a href="#contact" class="nav-link">Contact</a>
+            </nav>
+        </header>
+        <section id="about">
+            <video src="https://youtu.be/3I7PadT8czU" id="video"></video>
+        </section>
+        <section id="contact">
+            <form action="https://www.freecodecamp.com/email-submit" id="form">
+                <input type="email" name="email" id="email" Placeholder="Your Email Adress">
+                <input type="submit" id="submit">
+            </form>
+        </section>
```

Comment:
Please  enclose this block in a container such as <code>main</code> because is more semantic rather than having many sections itself